build.rs

```diff
@@ -18,22 +19,26 @@ fn main() {
 // -----------------------------------------------------------------------------
 // Solidity signatures
 // -----------------------------------------------------------------------------
-const SIGNATURE_FILES: [&str; 5] = [
-    "static/contracts/BRLCToken.signatures",
-    "static/contracts/CardPaymentProcessor.signatures",
-    "static/contracts/CashbackDistributor.signatures",
-    "static/contracts/PixCashier.signatures",
-    "static/contracts/TokenDistributor.signatures",
-];
+const SIGNATURES_DIR: &str = "static/contracts/*.signatures";
```

Comment:
Automatically loading all signatures files in the specified folder, so we can have the function names in logs and metrics.